### DISCUSSION TITLE
servers dont connect automatically

### DISCUSSION THREAD

**Discussion by emerymastering** [apohl79/audiogridder#951] [General]
Hello,

with beta 5 all was fine and servers were connected automatically. Now If I start both computers then server on other computer is not found in the list.
I need to restart server app and then plugin notices it. After that all other sessions are ok. But next day same thing again.
So somehow it doesn't connect on first run.  same in beta8 and 9.

Any idea why so? i am on mac on both systems.

Cheers
D

**Comment by apohl79** [comment]
Can you provide server and plugin logs?

**Comment by emerymastering** [comment]
[Server.zip](https://github.com/apohl79/audiogridder/files/8727212/Server.zip)
[FX.zip](https://github.com/apohl79/audiogridder/files/8727213/FX.zip)
Attaching. Maybe its coz IP changes every time I start the systems. I need to do manual connect, before was automatic.

**Comment by emerymastering** [comment]
After trying all versions I am back to beta5, works most stable and reliable for me. New versions introduced problems unfortunately.

**Reply by apohl79** [reply]
can you be more specific? whats not working? I have only two remaining known issues on my list. cheers

**Comment by emerymastering** [comment]
For me beta 5 loads quick and all is smooth. all later versions either don't connect automatically, server disappears from the list in plugin, so I have to restart it and then it appears. Weird stuttering happens which is gone after i change buffer size.
So I choose comfort and stability of beta 5 over new versions which should be improvement, but I feel no improvements, only troubles.
I am up for helping you to find out why this happens. I sent you the logs before, did they show anything useful?

Cheers
Denis

**Reply by apohl79** [reply]
yes that would be great. if you have an issue with a matching set of log/trace files, i can start to investigate. thx!

**Reply by apohl79** [reply]
btw, the connection issue that you have reported should be sorted in the next version.

**Comment by emerymastering** [comment]
looking forward, that would be major update if connections works as in beta5:)
